gap/Parser.gi

```diff
@@ -62,11 +62,7 @@ InstallGlobalFunction( AutoDoc_Type_Of_Item,
         entries := [ "Oper", "methods" ];
         has_filters := "List";
     elif PositionSublist( type, "DeclareConstructor" ) <> fail then
-        ## FIXME: there should be a Constructor tag, but it is unfortunately not possible, since GAPDoc
-        ##        does not offer such a tag. Issue for this is filed here:
-        ##        https://github.com/frankluebeck/GAPDoc/issues/23
-        ##        Once this is fixed, the next line needs to be changed accordingly.
-        entries := [ "Oper", "methods" ];
+        entries := [ "Constr", "methods" ];
```

Comment:
If you are eager to merge this now, perhaps do something as in this pseudo code:
```
  if GAPDoc >= 1.6.1 then
      entries := [ "Constr", "methods" ];
  else
     Print("Warning: Installed GAPDoc version does not support constructors");
     entries := [ "Oper", "methods" ];
  fi;
```
or even change the `Print` to an `Error`. In either case, you can drop the requirement for GAPDoc 1.6.1 from `PackageInfo.g`

Reply:
Currently, it is fine as it is. It prints constructors as Operations, as it was before. So there is no need to merge this until the new GAPDoc is released.